man/tpm2_loadexternal.1.md

```diff
@@ -0,0 +1,72 @@
+tpm2_loadexternal 1 "SEPTEMBER 2017" tpm2-tools
+==================================================
+
+NAME
+----
+
+tpm2_loadexternal(1) - load an object that is not a Protected Object into the
+TPM.
+
+SYNOPSIS
+--------
+
+`tpm2_loadexternal` [OPTIONS]
+
+DESCRIPTION
+-----------
+
+tpm2_loadexternal(1) - load an object that is not a Protected Object into the
+TPM. The command allows loading of a public area or both a public and a
+sensitive area.
+
+OPTIONS
+-------
+
+  * `-H`, `--hierarchy`=_HIERARCHY_:
+    hierarchy to use for the ticket.
+    Supported options are:
+      * `o` for `TPM_RH_OWNER`
+      * `p` for `TPM_RH_PLATFORM`
+      * `e` for `TPM_RH_ENDORSEMENT`
+      * `n` for `TPM_RH_NULL`
+
+  * `-u`, `--pubfile`=_PUBLIC\_FILE_:
+    The public portion of the object.
+
+  * `-r`, `--privfile`=_PRIVATE\_FILE_:
+    The sensitive portion of the object, optional.
+
+  * `-C`, `--context`=_CONTEXT\_FILE_
+    The file to save the object context, optional.
+
+[common options](common/options.md)
+
+[common tcti options](common/tcti.md)
+
+EXAMPLES
+--------
+
+Load a public area generated by tpm2_create, tpm2_readpublic or manually
+
+```
+tpm2_loadexternal -H <e|o|p|n> -u <pubKeyFileName> -C object.context
```

Comment:
Here again the example is inconsistent with the other man pages. The options are listed above so it makes no sense to list here again, but yes I agree with you that we should fix as a separate issue.